spark/src/main/java/org/apache/iceberg/actions/RewriteDataFilesAction.java

```diff
@@ -32,6 +32,10 @@
 import org.apache.spark.broadcast.Broadcast;
 import org.apache.spark.sql.SparkSession;
 
+/**
+ * @deprecated since 0.12.0, will be removed in 0.13.0; use {@link RewriteDataFilesAction} instead.
+ */
+@Deprecated
```

Comment:
Hi, should we use `BaseRewriteDataFilesAction` instead? `RewriteDataFilesAction` is the deprecated class itself.